kolibri/core/assets/src/api_resource.js

```diff
@@ -281,7 +281,11 @@ class ResourceManager {
       throw new TypeError('A resource with that name has already been registered!');
     }
     this._resources[name] = new ResourceClass(this._kolibri);
-    Object.defineProperty(this, ResourceClass.name, { value: this._resources[name] });
+    // Shim for IE9 compatibility of .name property.
+    // Modified from: http://matt.scharley.me/2012/03/monkey-patch-name-ie.html#comment-551654096
+    const className = ResourceClass.name || /function\s([^(]{1,})\(/.exec(
```

Comment:
`.name` is a very ES6 thing that does not get transpiled by Babel. This shim gives us consistent naming in IE9 too.